values.yaml

```diff
@@ -0,0 +1,45 @@
+clusterDomain: "se6099.opencraft.hosting"
```

Comment:
It's unclear to me if this values file should be here or the defaults make sense to me, specifically the opencraft domains and emails, could you explain your logic for adding them and/or what happens if `installCRDs` is not set to true?

Reply:
Actually, this whole file should not exist. It was committed accidentally.